README.md

```
@@ -134,34 +152,68 @@ For more details, run `holmes investigate <source> --help`
+   - If using the CLI: Use `-r` flag with [custom runbook files](./examples/custom_runbooks.yaml) or add to `~/.holmes/config.yaml`
 
-You can save settings in a config file to avoid passing them to the CLI each time:
+You can save common settings and API Keys in a config file to avoid passing them from the CLI each time:
 
 <details>
 <summary>Reading settings from a config file</summary>
 
-You can customize HolmesGPT's behaviour with command line flags, or you can save common settings in config file for re-use.
+You can save common settings and API keys in config file for re-use. Place the config file in <code>~/.holmes/config.yaml`</code> or pass it using the <code> --config</code>
 
 You can view an example config file with all available settings [here](config.example.yaml).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)